framework/components/ecl-buttons/_buttons.scss

```diff
@@ -1,354 +1,161 @@
-/*
-Buttons
```

Comment:
Pointer cursor missing on hover, not sure if this is desired behavior.

![buttons_hover](https://cloud.githubusercontent.com/assets/1923476/26545052/8750578a-4464-11e7-89c8-0d28cbccac25.gif)